src/v/archival/ntp_archiver_service.cc

```diff
@@ -665,7 +665,9 @@ ntp_archiver::schedule_uploads(model::offset last_stable_offset) {
       manifest().get_last_uploaded_compacted_offset());
 
     std::vector<upload_context> params;
-    if (_partition->get_ntp_config().is_compacted()) {
+    if (
+      config::shard_local_cfg().cloud_storage_enable_compacted_topic_reupload
```

Comment:
Need `()` after property name to get value

Reply:
fixed